ui/components-react/src/components-react/filter-builder/FilterBuilderRuleGroup.scss

```diff
@@ -0,0 +1,82 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
There are gaps in between rule group hover areas, which results in parent group being momentarily highlighted when moving the mouse between sibling groups. At a quick glance, it looks like this can be fixed by not using margins.